programs/locking/src/lib.rs

```diff
@@ -196,27 +158,19 @@ mod locking {
                 },
                 signer,
             ),
-            unmint_amount,
+            unlock_amount,
         )?;
 
-        emit!(EventUnStake {
-            unmint_amount,
-            token_amount: withdraw_amount,
+        emit!(EventUnlock {
+            unmint_amount: unlock_amount,
+            token_amount: unlock_amount,
         });
 
         Ok(())
     }
-
-    /// Change vault funder. Funder can deposit fund to the vault.
-    pub fn change_funder(ctx: Context<FunderChange>) -> Result<()> {
-        let vault = &mut ctx.accounts.vault;
-        let funder = *ctx.accounts.funder.key;
-        vault.funder = funder;
-        Ok(())
-    }
 }
 
-/// Accounts for [InitializeVault](/staking/instruction/struct.InitializeVault.html) instruction
+/// Accounts for [InitializeVault](/locking/instruction/struct.InitializeVault.html) instruction
```

Comment:
Pls update vault space, it is pretty small now

Reply:
Fixed